t/api/init-fini.c

```diff
@@ -19,11 +19,14 @@ int api_initialize_test(char* unifyfs_root,
 {
     diag("Starting API initialization test");
 
-    int n_configs = 1;
-    unifyfs_cfg_option chk_size = { .opt_name = "logio.chunk_size",
-                                    .opt_value = "32768" };
+    int n_configs = 2;
+    unifyfs_cfg_option options[2];
+    options[0].opt_name = "logio.chunk_size";
+    options[0].opt_value = "32768";
+    options[1].opt_name = "client.node_local_extents";
```

Comment:
I would prefer not to set this option in the API init test.

Reply:
I fixed this check my comment below